web/views/FcRequestsTrack.vue

```diff
@@ -397,7 +380,7 @@ export default {
       });
     },
     actionDownload(studyRequests) {
-      const rows = Array.prototype.concat.apply([], studyRequests.map(getItemRows));
+      const rows = Array.prototype.concat.apply([], studyRequests.map(getItemRow));
```

Comment:
just do `const rows = studyRequests.map(getItemRow);` - we're no longer returning multiple rows per study request (another thing that becomes simpler / clearer!)